index.html

```diff
@@ -118,7 +118,7 @@ <h2 id="stories-header1">Tonic</h2>
           <div class="hidden2"><button type="button">See Project</button></div>
         </div>
         <div class="tonic">
-          <img src="images/snap3.png" alt="snap" />
+          <img src="images/Snapshoot-Portfolio-4.svg" alt="snap" />
```

Comment:
Kindly update the alt text so that it conveys the purpose of the respective image as described above 😃